packages/pn-personafisica-webapp/src/redux/notification/__test__/reducers.test.ts

```diff
@@ -383,6 +390,68 @@ describe('Notification detail redux state tests', () => {
     expect(actualState).toStrictEqual(newState);
   });
 
+  it('Every payment should have recipientIdx', async () => {
```

Comment:
This test case and the next can be mooved to the test case "Should be able to fetch the notification detail"